it/resources/benchmark-in-memory-it.ini

```diff
@@ -20,7 +20,7 @@ datastore.type = in-memory
 
 
 [workloads]
-default.url = https://github.com/opensearch-project/opensearch-benchmark-workloads
+default.url = https://github.com/martin-gaievski/opensearch-benchmark-workloads
```

Comment:
Not sure the intent here was to change this in the mainline repository.  To switch to your fork locally, please follow [these instructions](https://github.com/opensearch-project/opensearch-benchmark-workloads?tab=readme-ov-file#testing-the-workload).